### ISSUE TITLE
[update] Initial Setup Video for 2023 Server One

### ISSUE THREAD

**Issue by Dominion5254** [Start9Labs/documentation#446]
Initial Setup video indicates to the user they should be waiting on the bep and chime sounds which are no longer correct for the 2023 form factor.

Video should be removed or updated to avoid confusion for users setting up a 2023 Server One.

**Comment by kn0wmad** [comment]
Video also mentions embassy.local instead of start.local.  Maybe we should yank it?

**Comment by gStart9** [comment]
Agreed, I just had a customer jump straight to this video at the top of the guide and get confused.

tiny PR to yank: #457
